src/Components/sidebar/sidebar.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import  "./sidebar.css"
+function Sidebar() {
+     return (
+    <div>
+          <div className="side-bar">
+            <div class="side-bar-heading">
+                <div>
+                    <h1>Filter</h1>
+                </div>
+                <div>
+                    <h1 onClick={() => {
+               
+                filterDispatch({ type: "CLEAR" });
```

Comment:
from where exactly is the filterDispatch coming from? 
There is no import of it anywhere in this file. and there is no mention of filterDispatch in any other file!

Reply:
yeah firstly i was trying to add cart-context in this pr only but i removed that and for that i will create another pr